style.css

```diff
@@ -0,0 +1,456 @@
+    color: #FFFBFE;
+}
+
+/*PORTFOLIO SECTION*/
+
+.portfolio {
+    color: #000000;
+    display: block;
+}
+
+.portfolio h3{
+    text-align: center;
+    letter-spacing: 6px;
+    font-size: 48px;
+    color: #565254;
+    font-weight: normal;
+}
+
+.portfolio p{
+    padding: 0 30px;
+}
+
+.projectBox{
+    text-align: center;
+    width: 33.3%;
+    height: 330px;
+    float: left;
+    text-shadow: -1px -1px 0 #000, 1px -1px 0 #000, -1px 1px 0 #000, 1px 1px 0 #000;
+    opacity: 0.5;
+    transition: all 0.3s;
+}
+
+.projectBox h4{
+    font-size: 48px;
+    letter-spacing: 5px;
+    padding-top: 5%;
+    color: #FFFFFF;
+    display: none;
+}
+
+.projectBox h3{
+    font-size: 48px;
+    letter-spacing: 5px;
+    color: #FFFFFF;
+    display: none;
+    font-weight: bold;
+    padding-top: 15%;
+}
+
+.projectBox P{
+    font-size: 24px;
+    letter-spacing: 2px;
+    font-weight: bold;
+    color: #FFFFFF;
+    display: none;
+}
+
+.projectBox:hover{
+    opacity: 1;
+    transform: scale(0.9);
+}
+.projectBox:hover h3{
+    display: block;
+}
+
+.projectBox:hover h4{
+    display: block;
+}
+
+.projectBox:hover p{
+    display: block;
+}
+
+.projectBox a:visited{
+    color: #FFFFFF;
+    text-decoration: none;
+}
+
+.projectBox .projectDone h4{
+    margin-top: 35%;
+}
+.project1{
+    background-image: url(images/pilotshop.png);
+    background-size: cover;
+    background-position: center;
+}
+
+.project2{
+    background-image: url(images/maydenlogo.png);
+    background-size: contain;
+    background-position: center;
+    background-repeat: no-repeat;
+}
+
+.project3{
+    background-image: url(images/workprogress1.jpg);
+    background-size: cover;
+    background-position: center;
+}
+
+.project4{
+    background-image: url(images/workprogress2.jpg);
+    background-size: cover;
+    background-position: center;
+}
+.project5{
+    background-image: url(images/workprogress3.jpg);
+    background-size: cover;
+    background-position: center;
+}
+.project6{
+    background-image: url(images/workprogress4.jpg);
+    background-size: cover;
+    background-position: center;
+}
+
+.boxes{
+    overflow: auto;
+}
+
+/*ABOUT ME*/
+
+.aboutMe{
+    text-align: center;
+    overflow: auto;
+    margin-bottom: 50px;
+}
+
+.aboutMe h2{
+    font-size: 48px;
+    color: #565254;
+    font-weight: normal;
+    letter-spacing: 6px;
+}
+
+.aboutMe p{
+    letter-spacing: 2px;
+    line-height: 18px;
+    color: #565254;
+}
+
+.aboutMe .archWorks{
+    font-size: 24px;
+}
+
+.archBox{
+    width: 33.3%;
+    height: 250px;
+    float: left;
+    border: 1px solid black;
+}
+
+/*CONTACT ME*/
+
+footer{
+    text-align: center;
+    background-color: #272727;
+    width: 100%;
+    height: 160px;
+    color: #747474
+}
+
+footer h2{
+    margin-bottom: 10px;
+}
+
+footer p{
+    margin: 10px;
+}
+
+footer .contact{
+    padding-top: 5px;
+}
+
+footer a{
+    text-decoration: none;
+    color: #FFFFFF;
+    opacity: 0.8;
+}
+footer .backUp{
```

Comment:
none of this stuff is doing anything

Reply:
responsible for the border effect on hover. border is currently on transparent in this block of code.